bitcoin_transfer/spend_your_coin.md

```diff
@@ -83,60 +83,65 @@ Do you remember the main questions? **From where, to where and how much?**
 Constructing the **TxIn** and adding it to the transaction is the answer to the "from where" question.  
 Constructing the **TxOut** and adding it to the transaction is the answer to the remaining ones.
 
-The donation address of this book is: [1KF8kUVHK42XzgcmJF4Lxz4wcL5WDL97PB](https://blockchain.info/address/1KF8kUVHK42XzgcmJF4Lxz4wcL5WDL97PB)  
-This money goes into my "Coffee and Sushi Wallet" that will keep me fed and compliant while writing the rest of the book.  
-If you succeed in completing this challenge you will be able to find your contribution amongst the **Hall of the Makers** on [http://n.bitcoin.ninja/](http://n.bitcoin.ninja/) \(ordered by generosity\).
+> The donation address of this book is: [1KF8kUVHK42XzgcmJF4Lxz4wcL5WDL97PB](https://blockchain.info/address/1KF8kUVHK42XzgcmJF4Lxz4wcL5WDL97PB)  
+This money goes into Nicolas' "Coffee and Sushi Wallet" that will keep him fed and compliant while writing the rest of the book.  
+If you succeed in completing this challenge on the MainNet you will be able to find your contribution among the **Hall of the Makers** on [http://n.bitcoin.ninja/](http://n.bitcoin.ninja/) \(ordered by generosity\).
 
+To get the our MainNet address:
 ```cs
 var hallOfTheMakersAddress = new BitcoinPubKeyAddress("1KF8kUVHK42XzgcmJF4Lxz4wcL5WDL97PB");
 ```
 
-If you are working on the testnet, send the testnet coins to any testnet address.
+Or if you are working on the TestNet, send the TestNet coins to [our TestNet address](https://live.blockcypher.com/btc-testnet/address/mzp4No5cmCXjZUpf112B1XWsvWBfws5bbB/).
 
 ```cs
-var hallOfTheMakersAddress = BitcoinAddress.Create("mzp4No5cmCXjZUpf112B1XWsvWBfws5bbB");
+var hallOfTheMakersAddress = BitcoinAddress.Create("mzp4No5cmCXjZUpf112B1XWsvWBfws5bbB", Network.TestNet);
 ```
 
 ### How much?
 
-If you want to send **0.5 BTC** from a **transaction input** with **1 BTC** you actually have to spend it all!  
-As the diagram shows below, your **transaction output** specifies  **0.5** BTC to Hall of The Makers and **0.4999** back to you.  
-What happens to the remaining **0.0001 BTC**? This is the miner fee in order to incentivize them to add this transaction into their next block.
+Bitcoin has [several units to use](https://en.bitcoin.it/wiki/Units). 1 bitcoin (BTC) is 1'000 milli-bitcoins/millibits. 1 milli-bitcoin (mBTC) is 1'000 micro-bitcoins/bits. And finally 1 (&micro;BTC) micro-bitcoin is 100 satoshis. 1 satoshi (sat) is the smallest unit on the Bitcoin network.  
+
+If you want to send **400 &micro;BTC** (a few dollars) from an **unspent output**, which holds **1'000 &micro;BTC**, you actually have to spend it all!  
```

Comment:
Same comments to these lines as above and below.

Reply:
Yesterday @NicolasDorier lowered the level of entry for the Hall of Makers to 0.0004 BTC. To make the sample easier to follow I think the uBTC is justifiable here, because otherwise the reader ends up counting zeros again and again.

Reply:
Not uBTC, bits.